mos-platform/rp6502/init-cpu.s

```diff
@@ -0,0 +1,7 @@
+; Initialize the CPU.
```

Comment:
I don't see `_start` referenced anywhere in this PR; what's the platform's startup procedure like?

Reply:
There's no startup other than CLD and TXS for the CPU.

Reply:
It just occurred to me to mention that the 6502 reset vector is 0x0200. No code runs before LLVM-MOS code.

Reply:
In that case, there should be an assert in the linker script that _start is 0x200. The script as is should guarantee this for the most part, but it should fail if violated. (This could happen if a text segment included data with very high alignment requirements.)

Reply:
I added the origin address and start address to the output_format and changed my tooling to use them.